test/fhirtypes/ElementDefinition.test.ts

```diff
@@ -979,6 +986,28 @@ describe('ElementDefinition', () => {
       );
       expect(localSlice).toBeDefined();
     });
+
+    it('should not throw an error when unfolding a slice and the sliced element is not found', () => {
+      const telecom = practitioner.elements.find(e => e.path === 'Practitioner.telecom');
+      telecom.slicing = {
+        ordered: false,
+        rules: 'open',
+        discriminator: [{ type: 'value', path: 'system' }]
+      };
+      const telecomSlice = telecom.addSlice('FooSlice');
+      // Rather than get a StructureDefinition with incorrect slicing,
+      // simulate the effect of the incorrect slicing: getting nothing back from slicedElement()
+      slicedElementSpy.mockReturnValueOnce(null);
```

Comment:
It looks to me like `practitioner` is refreshed on every test (in `beforeEach`). Wouldn't it be more straightforward (and realistic) to just _delete_ the telecom element directly from the `practitioner` SD after you create the slice?